<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listAuthors">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listBook">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AuthorsBornInXY">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Авторы рождённые в промежутке 1800 - 1890 годов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AythorsWithNBooks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Все авторы с количеством книг больше 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RussianBooks">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Все книги русских авторов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CountPages">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Все книги с количеством страниц больше 100</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="Add">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAuthor"/>
    <addaction name="actionBook"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionjson"/>
    <addaction name="actionXML"/>
   </widget>
   <widget class="QMenu" name="Htubcnhfwbzgjkmpjdfntkz">
    <property name="title">
     <string>Create Base</string>
    </property>
    <addaction name="actionCreate"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuSave"/>
   <addaction name="Htubcnhfwbzgjkmpjdfntkz"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionBook">
   <property name="text">
    <string>Book</string>
   </property>
  </action>
  <action name="actionjson">
   <property name="text">
    <string>json</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
